src/app/public/modules/progress-indicator/progress-indicator-reset-button/progress-indicator-reset-button.component.ts

```diff
@@ -1,51 +1,68 @@
 import {
-  Component,
-  Input,
   ChangeDetectionStrategy,
-  Output,
+  ChangeDetectorRef,
+  Component,
   EventEmitter,
-  OnDestroy
+  Input,
+  OnDestroy,
+  Output
 } from '@angular/core';
 
-import {
-  SkyProgressIndicatorComponent
-} from '../progress-indicator.component';
 import {
   SkyProgressIndicatorMessageType
 } from '../types';
 
+import {
+  SkyProgressIndicatorComponent
+} from '../progress-indicator.component';
```

Comment:
Same response as above. The imports from `../types` should be above the "local" imports because they are a level removed from the current source file.

Reply:
Ok. That one's new to me!

Reply:
Alphabetical ordering ☝️